src/main/resources/models/tables/Player.hjson

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
Lets remove this file as its quite similar to playerStatsView. Based on one of the earlier comments, we should keep the example code repo simple.